modules/n1ql/pages/n1ql-language-reference/stringfun.adoc

```diff
@@ -1042,3 +1042,45 @@ SELECT UPPER("N1QL is awesome") as n1ql;
 }
 ----
 ====
+
+[#fn-str-upper]
+== URLENCODE(plain_string)
+
+=== Description
+Returns the input string encoded for use in a URL
+
+=== Arguments
+in_str:: A string, or any valid xref:n1ql-language-reference/index.adoc[expression] which evaluates to a string, that is the string to convert to upper case.
+
+=== Return Value
+A string representing the input string converted to upper case.
```

Comment:
Fix description. Also we need to cover the common edge cases &mdash; although I admit this page is not very good at that!

```suggestion
The input string, with any reserved characters replaced by percent encoding.

If the input argument is MISSING, returns MISSING.
If the input argument is non-string, returns NULL.
```